src/main/kotlin/no/nav/familie/ef/mottak/service/PdfKvitteringService.kt

```diff
@@ -39,7 +39,7 @@ class PdfKvitteringService(
         when (innsending.dokumenttype) {
             DOKUMENTTYPE_OVERGANGSSTØNAD -> {
                 val dto = SøknadMapper.toDto<SøknadOvergangsstønad>(innsending)
-                SøknadTreeWalker.mapOvergangsstønad(dto, vedleggTitler)
```

Comment:
alle her burde vell være SøknadTilGenereltFormatMapper og ikke SøknadTreeWalker

Reply:
Enig, men vi har ikke laget det for de to andre ennå